src/ngx_pagespeed.cc

```diff
@@ -492,7 +492,8 @@ void copy_response_headers_from_ngx(const ngx_http_request_t* r,
 
   // When we don't have a date header, set one with the current time.
   if (headers->Lookup1(HttpAttributes::kDate) == NULL) {
-    headers->SetDate(ngx_current_msec);
```

Comment:
I did check that we're not using it anywhere else. I don't think a comment would be much use, though something to actually hide/break the variable wouldn't be an awful idea. Not sure exactly how to go about that, though.